### PR TITLE
BUG: Deselect previous item selection when dragging to select items

### PR DESCRIPTION
**Pull request by bergice**
This fixes the following:

1. Select some files/folders
2. Drag to select some files/folders
3. Notice that the previous selection is still selected

### CODE REVIEW

File: client/src/containers/Gallery/Gallery.js [hunk 1]
```diff
@@ -512,6 +512,8 @@ class Gallery extends Component {
    * @param event Event
    */
   handleGroupSelect(items, event) {
+    this.props.actions.gallery.deselectFiles();
```
Comment (robbieaverill) [inline client/src/containers/Gallery/Gallery.js]: Any reason you haven't included this in the destructuring on the next line?

Comment (bergice) [reply client/src/containers/Gallery/Gallery.js]: What do you mean?

Comment (bergice) [reply client/src/containers/Gallery/Gallery.js]: Just realised this should work with the concatenation logic below.

Comment (robbieaverill) [reply client/src/containers/Gallery/Gallery.js]: For reference, I meant add it to `const { foo, bar, deselectFiles } = this.props;` then call `deselectFiles()` directly, the destructuring from `this.props.actions.gallery` is already on the next line, so you'd just be adding another item to it

**Pull request merged**